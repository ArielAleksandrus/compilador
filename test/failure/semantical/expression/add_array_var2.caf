programa{
	car c[20], d[20];

	d = c + "ariel";
}