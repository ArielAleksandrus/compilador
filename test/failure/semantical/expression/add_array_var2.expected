ERROR => Line 4, col 6: Both lval and rval should be of type int or car. Found: CAR ARRAY and CAR ARRAY
